lib/steps/setupRedirects.js

```diff
@@ -44,7 +44,16 @@ const setupRedirects = () => {
     // One route may map to multiple Netlify routes: e.g., catch-all pages
     // require two Netlify routes in the _redirects file
     getNetlifyRoutes(route).map((netlifyRoute) => {
-      const redirect = `${netlifyRoute}  ${nextRedirect.target}  200`;
+      const { conditions, force, statusCode, target } = nextRedirect;
+      const redirectPieces = [
+        netlifyRoute,
+        target,
+        `${statusCode || "200"}${force ? "!" : ""}`
+      ];
+      if (conditions && conditions.length > 0) {
+        redirectPieces.push(conditions.join("  "));
+      }
```

Comment:
```suggestion
```

(in case the above is possible, where we just add conditions in the pieces assignment)